app/services/task-service.ts

```diff
@@ -89,6 +95,25 @@ export class TaskService extends ServiceBase {
         return getOnceProxy(this.get(jobId, taskId, options));
     }
 
+    public getMultiple(jobId: string, taskIds: string[], properties?: string): Observable<Task[]> {
```

Comment:
Here I think you can use the RxListProxy instead with the fetchAll() method instead